tests/scripts/thread-cert/Cert_8_1_01_Commissioning.py

```diff
@@ -65,7 +65,7 @@ def test(self):
 
         self.nodes[JOINER].interface_up()
         self.nodes[JOINER].joiner_start('openthread')
-        time.sleep(10)
+        time.sleep(15)
```

Comment:
Here are the Windows logs ([windows_logs.zip](https://github.com/openthread/openthread/files/541948/windows_logs.zip)) for the test. A few of the important lines:

```
07:14:36.221    127C    [otNodeApi]    ---> otNodeJoinerStart [2] openthread <NULL>
07:14:36.221    17EC       [otCore]    ---> Thread::MeshCoP::Joiner::Start
07:14:36.221    17EC       [otCore]    MLE Sent discovery request
07:14:36.222    1668       [otCore]    MLE Received discovery request
07:14:36.222    1668       [otCore]    MLE Sent discovery response
07:14:36.222    17EC       [otCore]    MLE Handle discovery response
07:14:40.578    17EC       [otCore]    COAP DTLS started
07:14:48.900    1668       [otCore]    COAP Generated KEK
07:14:48.901    17EC       [otCore]    COAP Sent joiner finalize
07:14:48.902    1668       [otCore]    COAP receive joiner finalize 1
07:14:48.903    1668       [otCore]    COAP Received kek
07:14:48.904    17EC       [otCore]    COAP join success!
07:14:51.230    127C    [otNodeApi]    ---> otNodeGetMasterkey [2]
07:14:51.230    127C    [otNodeApi]    <--- otNodeGetMasterkey
07:14:51.230    127C    [otNodeApi]    ---> otNodeGetMasterkey [1]
07:14:51.231    127C    [otNodeApi]    <--- otNodeGetMasterkey
```
I'm not exactly sure why the DTLS exchange is taking 8 seconds here.